policyengine_uk/tests/policy/baseline/gov/dwp/pension_credit/savings_credit/is_savings_credit_eligible.yaml

```diff
@@ -0,0 +1,70 @@
+- name: No one reached SPA before the cutoff year, ineligible for savings credit
+  period: 2023
+  input:
+    people:
+      person:
+        age: 68
+        is_male: true
+        meets_savings_credit_age_requirement: false
+    benunits:
+      benunit:
+        members: [person]
+        savings_credit_income: 10000
+  output:
+    is_savings_credit_eligible: false
+
+- name: Has pre-cutoff year SPA member and income above threshold (single), eligible
+  period: 2023
+  input:
+    people:
+      person:
+        age: 78
+        is_male: false
+        meets_savings_credit_age_requirement: true
+    benunits:
+      benunit:
+        members: [person]
+        relation_type: SINGLE
+        savings_credit_income: 10200
+  output:
+    is_savings_credit_eligible: true
+
+- name: One member reached SPA before cutoff year and income above threshold (couple), eligible
+  period: 2023
+  input:
+    people:
+      person:
+        age: 73
+        is_male: true
+        meets_savings_credit_age_requirement: true
+      spouse:
+        age: 66
+        is_male: false
+        meets_savings_credit_age_requirement: false
```

Comment:
We can take out these fields right- the age should be enough. Let's make sure it's calculating the `meets_savings_credit_age_requirement` correctly by avoiding giving it to the model for free.